reporter/html.js

```diff
@@ -676,20 +676,20 @@ QUnit.log( function( details ) {
 	// Also, we need to check details.expected existence, as it can exist and be undefined
 	if ( !details.result && hasOwn.call( details, "expected" ) ) {
 		if ( details.negative ) {
-			expected = escapeText( "NOT " + QUnit.dump.parse( details.expected ) );
+			expected = "NOT " + QUnit.dump.parse( details.expected );
 		} else {
-			expected = escapeText( QUnit.dump.parse( details.expected ) );
+			expected = QUnit.dump.parse( details.expected );
 		}
 
-		actual = escapeText( QUnit.dump.parse( details.actual ) );
+		actual = QUnit.dump.parse( details.actual );
```

Comment:
> breaking because the current behavior is broken

Unless the bug becomes a feature. :)

I'm +1 to merge this if we address the [test gap](https://github.com/jquery/qunit/pull/950#discussion_r55228353) mentioned by @gibson042